src/main/java/tutu/Tutu.java

```diff
@@ -0,0 +1,90 @@
+package tutu;
+
+import java.io.File;
+import java.io.IOException;
+import tutu.data.Storage;
+import tutu.data.TaskList;
+import tutu.exception.InvalidInputException;
+import tutu.task.*;
+import tutu.user.Parser;
+import tutu.user.Ui;
+
+/** Represents a todo list that takes in command inputs from the user. */
+public class Tutu {
+    /** TaskList object that stores the tasks added */
+    private static TaskList taskList = new TaskList();
+
+    /**
+     * Prints a line of asterisks to separate between user inputs and resulting outputs.
+     */
+    public static void separator() {
```

Comment:
According to the Java coding standard, names representing methods must be verbs. Perhaps you can add a verb to the prefix of your current method name such as print.
```suggestion
    public static void printSeparator() {
```